src/guillotine/fetchFromApi.ts

```diff
@@ -1,31 +1,23 @@
-import type {ContentApiBaseBody, GuillotineResponse, GuillotineResponseJson, ProjectLocaleConfig} from '../types';
-
+import type {FetchOptions, GuillotineResponse, GuillotineResponseJson, ProjectLocaleConfig} from '../types';
 
 /** Generic fetch */
 export async function fetchFromApi<Data = Record<string, unknown>>(
     apiUrl: string,
-    body: ContentApiBaseBody,
     projectConfig: ProjectLocaleConfig,
-    headers?: {},
-    method = 'POST',
-) {
-    const options = {
-        method,
-        headers: {
-            'Accept': 'application/json',
-            'Content-Type': 'application/json',
-            'X-Guillotine-SiteKey': projectConfig.site,
-        },
-        body: JSON.stringify(body),
-    };
+    options?: FetchOptions,
+): Promise<GuillotineResponseJson> {
 
-    if (headers) {
-        Object.assign(options.headers, headers);
-    }
+    const opts = {
```

Comment:
Where did the type go? :RequestInit

Reply:
It is not assignable to RequestInit type because of the `next` field that is added by Next